app/views/step_by_step_pages/show.html.erb

```diff
@@ -18,13 +18,15 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <% unless @step_by_step_page.steps_have_content? %>
+    <% unless @step_by_step_page.steps_have_content? && @step_by_step_page.links_checked? %>
       <%= render "govuk_publishing_components/components/inset_text" do %>
         <h2 class="govuk-heading-m">To publish this step by step you need to</h2>
         <% if !@step_by_step_page.steps.any? %>
           <%= link_to('Add at least one step', new_step_by_step_page_step_path(@step_by_step_page), class: 'govuk-link govuk-link--no-visited-state') %>
-        <% else %>
+        <% elsif !@step_by_step_page.steps_have_content? %>
           <%= link_to('Add content to all your steps', '#steps', class: 'govuk-link govuk-link--no-visited-state') %>
+        <% elsif !@step_by_step_page.links_checked? %>
+          <%= link_to "Check for broken links", { action: "check_links", step_by_step_page_id: @step_by_step_page.id}, method: :post, class: 'govuk-link govuk-link--no-visited-state' %>
```

Comment:
Why doesn't this do the same as "Add at least one step" and show link users to the steps section where the "Check for broken links" button is?

Reply:
I'll clarify with Tobi - I would have thought this should actually submit the 'Check for broken links' form.

Reply:
I'll clarify with Tobi - I would have thought this should actually submit the 'Check for broken links' form.

Reply:
Have double-checked, she agrees it should link to the section 👍 this may change after user research. I'll go ahead and do that now - simplifies it a lot!